source/includes/_tile.md

```diff
@@ -20,6 +20,50 @@ This endpoint identifies which lock belongs to the specific tile.
 
 Replace `TILE_ID` with the appropriate tile ID.
 
+## Get Lock Belonging To Tile (v2)
+
+```shell
+curl 'https://api.doordeck.com/tile/00000000-0000-0000-0000-000000000000/'
+  -H "Authorization: Bearer TOKEN"
+```
+
+> Replace `00000000-0000-0000-0000-000000000000` with the tile ID.
+
+> The above command returns 404 if no tile is known, or a see other 303 with the `Location` header set to the value of
+the lock
```

Comment:
```suggestion
```

Reply:
Normally we say something along the lines of "The above command returns JSON structured as follows: 
...JSON EXAMPLE OF RESPONSE..."